File: scripts/io.py
def output(message: str):
  print(message)
  return message

def read(path: str):
  with open(path, "r", encoding="utf8") as f:
    return f.read()

def write(path: str, text: str):
  with open(path, "w") as f:
    return f.write(text)

File: scripts/io.ts
import * as fs from 'fs'

function output(message: any) {
  console.log(message)
  return message
}

function read(path: string) {
  return fs.readFileSync(path, 'utf-8')
}

function write(path: string, text: string) {
  fs.writeFileSync(path, text)
  return text
}